src/Microsoft.Fx.Portability.Reports.Excel/ExcelOpenXmlOutputWriter.cs

```diff
@@ -152,7 +152,7 @@ private void GenerateSummaryPage(Worksheet summaryPage, ReportingResult analysis
                     var summaryData = new List<object>() { analysisResult.GetNameForAssemblyInfo(item.SourceAssembly), item.SourceAssembly.TargetFrameworkMoniker ?? string.Empty };
 
                     // TODO: figure out how to add formatting to cells to show percentages.
-                    summaryData.AddRange(item.UsageData.Select(pui => (object)((pui.PortabilityIndex == 0) ? 0 : double.Parse((pui.PortabilityIndex * 100).ToString("##.##")))));
+                    summaryData.AddRange(item.UsageData.Select(pui => (object)(Math.Round(pui.PortabilityIndex * 100.0, 2))));
```

Comment:
Not sure why it was formatted out to two decimals and then parsed back - I think rounding will do the same thing